packages/api/tests/lambdas/test-sqs-message-consumer.js

```diff
@@ -362,3 +363,63 @@ test.serial('SQS message consumer queues correct number of workflows for rules m
     await cleanupRulesAndQueues(rules, [queue]);
   });
 });
+
+test.serial('processQueues archives messages from the ENABLED sqs rule only', async (t) => {
+  const { rules, queues } = await createRulesAndQueues();
+  const message = { testdata: randomString() };
+
+  // Send message to ENABLED queue
+  const firstMessage = await SQS.sendSQSMessage(
+    queues[0].queueUrl,
+    message
+  );
+
+  // Send message to DISABLED queue
+  const secondMessage = await SQS.sendSQSMessage(
+    queues[1].queueUrl,
+    { testdata: randomString() }
+  );
+  const keyOne = `${process.env.stackName}/archived-incoming-messages/${firstMessage.MessageId}`;
+  const keyTwo = `${process.env.stackName}/archived-incoming-messages/${secondMessage.MessageId}`;
+
+  await handler(event);
+
+  const firstItem = await s3().getObject({
+    Bucket: process.env.system_bucket,
+    Key: keyOne,
+  }).promise();
+  const messageBody = JSON.parse(JSON.parse(firstItem.Body.toString()));
+  t.deepEqual(messageBody, message);
+
+  await t.throwsAsync(s3().getObject({
+    Bucket: process.env.system_bucket,
+    Key: keyTwo,
+  }).promise(), { code: 'NoSuchKey' });
+
+  t.teardown(async () => {
+    await cleanupRulesAndQueues(rules, queues);
+  });
+});
+
+test.serial('archiveMessage archives all SQS messages', async (t) => {
```

Comment:
What is this test specifically verifying vs the previous test? It says "archive all messages", perhaps implying that it should be testing that multiple messages get archived, but it's not setting up multiple messages? If it were, that might be a valuable test

Reply:
Oh, gotcha